src/components/Spinner/index.tsx

```diff
@@ -1,41 +1,5 @@
 import './index.css';
 
-import { useState } from 'react';
-import { useInterval } from 'usehooks-ts';
-
 export function Spinner() {
   return <div className="lds-dual-ring" />;
 }
-
-export function ProgressSpinner({ badge }: { badge: string }) {
```

Comment:
Sometimes we temporarily stop using utility components (like this progress bar) but we would use them again in the future (or maybe not!).

What should/can we do about them?

Reply:
I don't think we will have any actual progress percentage in the near future (only statuses like 'pending', 'done', etc.), so we probably won't use the `ProgressSpinner` component because that would just be "lying" to the users.

Reply:
That's a good point.

Reply:
I would strongly err on the side of deleting unused code. If we find we need it again, we can always bring it back from git history. But much more likely is that we don't and we're happy to have less code. I'd only keep dead code around if you know you're going to be using in the next few days.